src/cubicbez.rs

```diff
@@ -31,7 +31,7 @@ struct ToQuads {
 impl CubicBez {
     /// Create a new cubic Bézier segment.
     #[inline]
-    pub fn new<V: Into<Vec2>>(p0: V, p1: V, p2: V, p3: V) -> CubicBez {
+    pub fn new<P: Into<Point>>(p0: P, p1: P, p2: P, p3: P) -> CubicBez {
```

Comment:
this has the limitation of requiring all of these arguments to be of the same type. Assuming we like `Into<Point>`, there's a case that we might want to have each argument `impl Into<Point>` independently, so you could do things like `CubicBez::new(point_a, point_b, (1.0, 32.0), (0., 101.))`?